docs/supported_instruments.rst

```diff
@@ -0,0 +1,18 @@
+Supported Instruments
+=====================
+
+.. _cosmic_gps:
+
+COSMIC GPS
+----------
+
```

Comment:
Recommend adding a short summary here about COSMIC and providing a link to an information webpage.

Reply:
automodule takes care of that: https://pysatcdaac.readthedocs.io/en/rc_0_0_4/supported_instruments.html#module-pysatCDAAC.instruments.cosmic_gps